lib/Comment/styles.scss

```diff
@@ -32,3 +32,35 @@
     background: none;
   }
 }
+
+.new-comment__suggestions,
+.edit-comment__suggestions {
+  &__list {
+    @apply absolute overflow-y-scroll bg-white border border-solid border-neutral-90 rounded max-h-128 min-w-64;
+    top: $layout-spacing-base / 2 + ($layout-spacing-base / 4);
+    margin-bottom: $layout-spacing-base * 3 !important; // have to override inline styles from react-mentions package :(
```

Comment:
I would love to use `margin` instead of `top`, but react-mentions overrides it with their own inline styles...

Reply:
gross!